exonum-java-binding/core/rust/exonum-java/Cargo.toml

```diff
@@ -5,7 +5,6 @@ authors = ["Exonum team <contact@exonum.com>"]
 
 [dependencies]
 env_logger = "0.7"
-exonum-btc-anchoring = "0.13.0-rc.2"
```

Comment:
It never seems to stay here for long 🙃

Reply:
It doesn't even feel right to add a task to enable it later :D